<launch>
    <arg name="ns" default="" />
    <arg name="scan_topic" default="scan" />
    <node pkg="amcl" type="amcl" name="amcl">
        <remap from="scan" to="$(arg scan_topic)" />
        <param name="odom_frame_id" value="$(arg ns)/odom" />
        <param name="base_frame_id" value="$(arg ns)/base_link" />
        <param name="global_frame_id" value="$(arg ns)/map" />
        <param name="use_map_topic" value="false" />
        <param name="odom_model_type" value="omni-corrected" />
        <param name="odom_alpha1" value="0.05" />
        <param name="odom_alpha2" value="0.05" />
        <param name="odom_alpha3" value="0.10" />
        <param name="odom_alpha4" value="0.05" />
        <param name="odom_alpha5" value="0.03" />
        <param name="min_particles" value="800" />
        <param name="max_particles" value="8000" />
    </node>
    <node pkg="rosservice" type="rosservice" name="global_loc" args="call --wait /global_localization" />
</launch>